.gitignore

```diff
@@ -81,7 +81,7 @@ typings/
 .nuxt
 
 # rollup.js default build output
-dist/
```

Comment:
Don't ever add dist to repo. If a file in here is really needed, it should be located elsewhere.

Reply:
maybe `examples`? That way they can `yarn build:dev`, and then go to `examples/demo.html` and have it refer to `../dist/...` in the path?

Reply:
Should be good to go now.